runtime/js/40_files.js

```diff
@@ -181,6 +185,11 @@
       }
       return this.#readable;
     }
+
+    setRaw(mode, options = {}) {
+      const rOptions = { ...DEFAULT_SET_RAW_OPTIONS, ...options };
+      ops.op_stdin_set_raw({ mode, options: rOptions });
```

Comment:
- Flatten arguments in the op 
- Optional: enable fast api with `#[op(fast)]`. just need to map errors to an enum-like thing with `#[repr(u32)]`

```rust
#[op(fast)]
fn op_set_raw(state: &mut OpState, rid: ResourceId, mode: bool, cbreak: bool) -> u32
```